src/components/v2/BorrowMarket/index.tsx

```diff
@@ -0,0 +1,152 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import BigNumber from 'bignumber.js';
+
+import {
+  convertWeiToCoins,
+  formatCoinsToReadableValue,
+  formatDollarsToReadableValue,
+} from 'utilities/common';
+import { TokenSymbol } from 'types';
+import { CONTRACT_TOKEN_ADDRESS } from 'utilities/constants';
+import { Token } from '../Token';
+import { Table, ITableProps } from '../Table';
+import { BorrowAsset } from './types';
+import { useStyles } from './styles';
+
+export * from './types';
+
+export interface IBorrowMarketUiProps {
+  className?: string;
+  borrowAssets: BorrowAsset[];
+}
+
+const columns = [
+  { key: 'asset', label: 'Asset', orderable: false },
+  { key: 'apy', label: 'APY', orderable: true },
+  { key: 'wallet', label: 'Wallet', orderable: true },
+  { key: 'liquidity', label: 'Liquidity', orderable: true },
+];
+
+export const BorrowMarketUi: React.FC<IBorrowMarketUiProps> = ({ className, borrowAssets }) => {
+  const styles = useStyles();
+
+  // Format assets to rows
+  const rows: ITableProps['data'] = borrowAssets.map(asset => [
+    {
+      key: 'asset',
+      render: () => <Token symbol={asset.symbol} />,
+      value: asset.name,
+    },
+    {
+      key: 'apy',
+      render: () => `${asset.borrowApyPercentage}%`,
+      value: asset.borrowApyPercentage,
+    },
+    {
+      key: 'wallet',
+      render: () =>
+        formatCoinsToReadableValue({
+          value: asset.walletBalanceCoins,
+          tokenSymbol: asset.symbol,
+        }),
+      value: asset.walletBalanceCoins.toString(),
+    },
+    {
+      key: 'liquidity',
+      render: () => formatDollarsToReadableValue(asset.liquidityCents),
+      value: asset.liquidityCents.toString(),
+    },
+  ]);
+
+  return (
+    <div className={className} css={styles.container}>
+      {/* TODO: add loading state */}
+
+      {/* TODO: add error state */}
+
+      <Table
+        title="Borrow market"
+        columns={columns}
+        data={rows}
+        initialOrder={{
+          orderBy: 'apy',
+          orderDirection: 'asc',
+        }}
+      />
+    </div>
+  );
+};
+
+const BorrowMarket: React.FC = () => {
+  // TODO: fetch actual data
+  const assetDataFromContract = [
```

Comment:
I would make this accessible to be used in tests and story books by exporting it from the module scope, placing it in its own file or in the storybook. This could be passes as a Prop as the UI component will receive it from a container component

Reply:
Yeah that's a good idea.

Reply:
On second thought, I just realized this will be specific to the contract itself. I don't know what the data will actually look like yet, I'll have more information on that once I get to connect the container to the contract.
When I get to it, I'll extract the test data so it can be reused as you suggested